src/imars3d/ui/stages/metadata.py

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env python3
-
+"""
+First stage to generate interactive reconstruction
+"""
 import param
 import panel as pn
 from pathlib import Path
+from imars3d.backend.io.config import save_config
 
 
 class MetaData(param.Parameterized):
-    # facility param required to locate the data on
-    # analysis mounts
+    # configuration to carry into the next stage
+    config_dict = param.Dict(
+        default={
+            "facility": "TBD",
```

Comment:
Shouldn't we be creating a `Parameter` for each of these entries so that we can add a type and validate them? For instance, `facility` could be only one of `"SNS", "HFIR"`. Similarly, `ipts` should be a positive integer